### PR TITLE
ENT - RMP-472 - bug fix - Added placeholder translation to autocompleteinput and updated unit tests

### PR DESCRIPTION
**Pull request by gayanSandamal**

### CODE REVIEW

**Review by Super-Chama — APPROVED** [review]
Please update the changelog.md

File: components/src/core/components/Input/Autocomplete/AutocompleteInput.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <oxd-autocomplete-text-input
       v-bind="$attrs"
       :clear="showClear"
-      :placeholder="placeholder"
+      :placeholder="$t(placeholder)"
```
Comment (Super-Chama) [inline components/src/core/components/Input/Autocomplete/AutocompleteInput.vue:6]: `translateMixin` is not imported? how is this working?

**Review by Super-Chama — CHANGES_REQUESTED** [review]

**Comment by gayanSandamal** [comment]
> 

Updated the changelog.md

File: components/src/core/components/Input/Autocomplete/AutocompleteInput.vue (hunk 1 above)
Comment (gayanSandamal) [reply components/src/core/components/Input/Autocomplete/AutocompleteInput.vue:6]: imported translateMixin

**Review by Super-Chama — APPROVED** [review]

**Pull request merged**